bazel/README.md

```diff
@@ -145,7 +145,7 @@ bazel build --config=libc++ //source/exe:envoy-static
 ```
 Note: this assumes that both: clang compiler and libc++ library are installed in the system,
 and that `clang` and `clang++` are available in `$PATH`. On some systems, you might need to
-include them in the search path, e.g. `export PATH=/usr/lib/llvm-8/bin:$PATH`.
+include them in the search path, e.g. `export PATH=/usr/lib/llvm-9/bin:$PATH`.
 
 You might also need to ensure libc++ is installed correctly on your system, e.g. on Ubuntu this
 might look like `sudo apt-get install libc++abi-8-dev libc++-8-dev`.
```

Comment:
do we want to recommend using v9 of libc++ as well?

Reply:
yes